changes/5129.added

```diff
@@ -0,0 +1 @@
+Implemented job list tile view.
```

Comment:
Let's call this out in `nautobot/docs/release-notes/version-2.2.md` as a significant new feature.